src/core/generator/clones/utils/cloning.ts

```diff
@@ -44,15 +61,25 @@ export const cloneIcon = async (
   return stringify(svg);
 };
 
-/** Gets the style attribute of an SVG node if it exists. */
+/**
+ * Gets the style attribute of an SVG node if it exists.
+ *
+ * @param {INode} node - The SVG node to get the style attribute from.
+ * @returns {Record<string, string>} The style attribute as an object.
+ */
 export const getStyle = (node: INode) => {
   if (node && node.attributes && node.attributes.style) {
     return parseStyle(node.attributes.style);
   }
   return {};
 };
 
-/** Parses the style attribute of an SVG node. */
+/**
+ * Parses the style attribute of an SVG node.
+ *
+ * @param {string} css - The style attribute as a string.
+ * @returns {Record<string, string>} The style attribute as an object.
```

Comment:
While scrolling threw the changes I noticed, that if we add the type information into the curly braces (`@returns {Record<string, string>}`), it will increase the effort to maintain this thing, once the return type changes right? I'm not sure if all IDEs will maintain the comments once we change the type in the code. Imagine you refactor the return type, but forget to update the comment. It will be more confusing because the comment gets outdated then.

Reply:
> if we add the type information into the curly braces ... it will increase the effort to maintain

I agree. JSDoc at that level was very useful in JavaScript, but TypeScript is already typed, so adding types to doc comments doesn't really add much value here.

Reply:
> > if we add the type information into the curly braces ... it will increase the effort to maintain
> 
> I agree. JSDoc at that level was very useful in JavaScript, but TypeScript is already typed, so adding types to doc comments doesn't really add much value here.

That is, it will be better to simply indicate parameter hints in the comments, and types in the functions themselves

Reply:
> > if we add the type information into the curly braces ... it will increase the effort to maintain
> 
> I agree. JSDoc at that level was very useful in JavaScript, but TypeScript is already typed, so adding types to doc comments doesn't really add much value here.

That is, it will be better to simply indicate parameter hints in the comments, and types in the functions themselves